createdisk.sh

```diff
@@ -97,12 +97,38 @@ if podman manifest inspect quay.io/crcont/routes-controller:${OPENSHIFT_VERSION}
     image_tag=${OPENSHIFT_VERSION}
 fi
 
+# create the tap device interface with specified mac address
+# this mac address is used to allocate a specific IP to the VM
+# when tap device is in use.
+${SSH} core@${VM_IP} 'sudo bash -x -s' <<EOF
+  nmcli connection add type tun ifname tap0 con-name tap0 mode tap autoconnect yes 802-3-ethernet.cloned-mac-address 5A:94:EF:E4:0C:EE
+EOF
```

Comment:
Isn't this equivalent to `${SSH} core@${VM_IP} 'sudo nmcli connection add type tun ifname tap0 con-name tap0 mode tap autoconnect yes 802-3-ethernet.cloned-mac-address 5A:94:EF:E4:0C:EE'`?

Reply:
In short; you are questioning why this needs to be wrapped in a `sudo bash -x -s`.
Would otherwise an error occur? I do not see characters that would be wrongly interpreted by the host shell (like zsh could do).

Reply:
this is not being changed for the self-sufficient bundle, its being with the existing situation which is that there is a container image which runs the `gvforwarder` and that container also has a dhcp client script which configures the interface using the dhcp service from gvisor-tap-vsock

Reply:
i think we can also scp the `NetworkManger` config file to `/etc/NetworkManager/system-connections` instead of running `nmcli` commands, there's a config file in: https://github.com/containers/gvisor-tap-vsock/blob/main/contrib/networkmanager/vsock0.nmconnection 

```
[connection]
id=tap0
type=tun
autoconnect=true
interface-name=tap0

[tun]
mode=2

[802-3-ethernet]
cloned-mac-address=5A:94:EF:E4:0C:EE

[ipv4]
method=auto

[proxy]
```